VideoWeb/VideoWeb.AcceptanceTests/Steps/WaitingRoomSteps.cs

```diff
@@ -366,6 +366,11 @@ private void CheckParticipantsAreStillConnected()
                     participant = "Panel Member";
                     panelElement = JudgeParticipantPanel.PanelMemberStatus(user.Id);
                 }
+                else if (user.HearingRole.Equals("Winger", StringComparison.OrdinalIgnoreCase))
```

Comment:
[GENERAL-COMMENT] Give a thought about changing to switch case instead of if else if.